data/default/default.lua

```diff
@@ -165,24 +181,23 @@ function _deflua_hut_frighten_callback(unit, extra)
 end
 signal.connect("hut_frighten", "_deflua_hut_frighten_callback")
 
---[[
-  Make partisans around conquered city
-
-  if requirements to make partisans when a city is conquered is fulfilled
-  this routine makes a lot of partisans based on the city`s size.
-  To be candidate for partisans the following things must be satisfied:
-  1) The loser of the city is the original owner.
-  2) The Inspire_Partisans effect must be larger than zero.
-
-  If these conditions are ever satisfied, the ruleset must have a unit
-  with the Partisan role.
-
-  In the default ruleset, the requirements for inspiring partisans are:
-  a) Guerilla warfare must be known by atleast 1 player
-  b) The player must know about Communism and Gunpowder
-  c) The player must run either a democracy or a communist society.
-]]--
-
+--- Make partisans around conquered city
+---
+--- if requirements to make partisans when a city is conquered is fulfilled
+--- this routine makes a lot of partisans based on the city`s size.
+--- To be candidate for partisans the following things must be satisfied:
+--- 1) The loser of the city is the original owner.
```

Comment:
Make this a numbered list with the `#.` in front instead of a number.

Reply:
This is also unordered, I don't know why they chose numbers.